readme: Running instructions (#298)
hearth-rs/hearth: README.md

hearth-server --help # prints usage info for the Hearth server
```

## Running with Kindling
The provided Hearth binaries act as a runtime and will do nothing on their own.
For Hearth to work properly you must give it a WebAssembly script to serve as

> the init system and a directory to serve as its root. Hearth provides a set of
> basic utilities for use with the runtime. These utilities are called Kindling
> and can be found in the [corresponding directory](./kindling). Kindling's root

Reviewer: Adapt how you want to fit the formatting you'd like.

```suggestion
the init system and a directory to serve as its root. Hearth provides a set of
batteries-included utilities for use with the runtime, named Kindling.
Kindling's source code is located within this repository [here](./kindling). Kindling's root
```

Author: making some bigger changes to this paragraph. Let me know what you think.

> and init system can be built with the script found in the tools directory. A
> cargo alias is provided for ease of use.

Reviewer: ```suggestion
and init system can be built with the script found in the tools directory.

A Cargo alias is provided for ease of use:
```

```sh
cargo build-root # build Kindling using the build script.
```

> Now that the root has been built, Hearth can be ran with kindling using the
> following command in the root of the repository.

Reviewer: ```suggestion
Now that the root has been built, Hearth can be ran with Kindling as its root:
```

Author: This was meant to state that this command only works when in the root of the repository.

Reviewer: The runtime's filesystem root vs the repository root is ambiguous, could you suggest a better way of conveying that?

Author: Honestly I think that my original clarification wasnt actually all that necessary.
